### PR TITLE
SCAL-219625: sage split changes to support TSE fullApp

### PR DESCRIPTION
**Pull request by mouryabalabhadra**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@thoughtspot/visual-embed-sdk",
-    "version": "1.33.0-alpha.1",
```
Comment (sastaachar) [inline package.json:3]: no need to update this , here alpha is fine

File: src/embed/app.ts [hunk 2]
```diff
@@ -401,6 +406,11 @@ export interface AppViewConfig extends Omit<ViewConfig, 'visibleTabs'> {
      * @version SDK : 1.29.0 | Thoughtspot : 10.1.0.cl
      */
     isOnBeforeGetVizDataInterceptEnabled?: boolean;
+    /**
+     * Flag to use home page search bar mode
+     * @version SDK : 1.33.0-alpha.2 | Thoughtspot : 10.2.0.cl
```
Comment (sastaachar) [inline src/embed/app.ts:411]: update sdk version here to 1.33

Comment (mouryabalabhadra) [reply src/embed/app.ts:411]: ack

File: package.json (hunk 1 above)
Comment (mouryabalabhadra) [reply package.json:3]: ack

**Review by sastaachar — APPROVED** [review]

**Pull request merged**